src/ray/gcs/gcs_server/actor_info_handler_impl.cc

```diff
@@ -0,0 +1,84 @@
+#include "actor_info_handler_impl.h"
+#include "ray/util/logging.h"
+
+namespace ray {
+namespace rpc {
+
+void DefaultActorInfoHandler::HandleAsyncGet(const rpc::ActorAsyncGetRequest &request,
+                                             rpc::ActorAsyncGetReply *reply,
+                                             rpc::SendReplyCallback send_reply_callback) {
+  RAY_LOG(DEBUG) << "Begin get actor info, actor id is:" << request.actor_id();
+  ActorID actor_id = ActorID::FromBinary(request.actor_id());
+  auto on_done = [request, reply, send_reply_callback](
+                     Status status, const boost::optional<ActorTableData> &result) {
+    if (status.ok()) {
+      if (result) {
+        reply->mutable_actor_table_data()->CopyFrom(*result);
+      }
+    } else {
+      RAY_LOG(ERROR) << "Failed to get actor info:" << status.ToString()
+                     << ",actor id is:" << request.actor_id();
+    }
+    send_reply_callback(status, nullptr, nullptr);
+  };
+
+  Status status = gcs_client_.Actors().AsyncGet(actor_id, on_done);
+  if (!status.ok()) {
+    on_done(status, boost::optional<ActorTableData>());
```

Comment:
Use `boost::none` instead of `boost::optional<ActorTableData>()`?

Reply:
ok, i have fix it.